Gemfile

```diff
@@ -0,0 +1,5 @@
+source 'https://rubygems.org'
+
+gem 'pronto'
+gem 'pronto-rubocop', require: false
+gem 'pronto-flay', require: false
```

Comment:
Bundler/OrderedGems: Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem `pronto-flay` should appear before `pronto-rubocop`.